### PR TITLE
[ENG-2458] Registration Moderation Email Fixes

### PR DESCRIPTION
**Pull request by UdayVarkhedkar**
## Purpose
Tweaks email templates to ensure they are up to our standards and communicate effectively

## Changes
* Added a missing period to `pending_registration_admin.html.mako`
* Added a 100px `max-height` attribute to the header logo image provided for providers in `notify_base.html.mako`
* Modifies `withdrawal_request_declined.html.mako` to be addressed to the correct user
* Adjusts `notification_settings_url` parameter in `notify_base.html.mako` depending on resource type
* Fix links for rejected registration in `reviews_submission_status.html.mako` and adds `draft_registration` to email template context
  * Modifies `test_registration_moderation_notifications.py` to expect new draft_registration context parameter

## QA Notes
Verify the bugs reported in Notion are properly addressed

## Documentation
N/A

## Side Effects
N/A

## Ticket
[JIRA Ticket](https://openscience.atlassian.net/browse/ENG-2458)

### CODE REVIEW

File: osf/utils/notifications.py [hunk 1]
```diff
@@ -18,12 +18,13 @@ def get_email_template_context(resource):
         'provider_support_email': resource.provider.email_support or OSF_SUPPORT_EMAIL,
         'document_type': document_type
     }
-
     if document_type == 'registration' and resource.provider.brand:
         brand = resource.provider.brand
         base_context['logo_url'] = brand.hero_logo_image
         base_context['top_bar_color'] = brand.primary_color
         base_context['provider_name'] = resource.provider.name
+    elif document_type == 'registration':
```
Comment (brianjgeiger) [inline osf/utils/notifications.py:26]: Drive-by: Should this be `if`? Otherwise it will only hit if it's an unbranded registration provider.

**Comment by brianjgeiger** [comment]
Test failure

```py
=================================== FAILURES ===================================
3187_ TestCreatePreprintRequestAction.test_moderator_can_reject_moderated_requests _
3188api_tests/requests/views/test_request_actions_create.py:359: in test_moderator_can_reject_moderated_requests
3189    res = app.post_json_api(url, payload, auth=moderator.auth)
3190tests/json_api_test_app.py:125: in wrapper
3191    return JSONAPIWrapper.make_wrapper(self, url, method, content_type, params, **kw)
3192tests/json_api_test_app.py:33: in make_wrapper
3193    wrapper = self._gen_request(method, url, **kw)
3194../../../virtualenv/python3.6.3/lib/python3.6/site-packages/webtest_plus/app.py:177: in _gen_request
3195    **kwargs)
3196../../../virtualenv/python3.6.3/lib/python3.6/site-packages/webtest/app.py:753: in _gen_request
3197    expect_errors=expect_errors)
3198tests/json_api_test_app.py:89: in do_request
3199    expect_errors)
3200../../../virtualenv/python3.6.3/lib/python3.6/site-packages/webtest/app.py:623: in do_request
3201    res = req.get_response(app, catch_exc_info=True)
3202../../../virtualenv/python3.6.3/lib/python3.6/site-packages/webob/request.py:1310: in send
3203    application, catch_exc_info=True)
3204../../../virtualenv/python3.6.3/lib/python3.6/site-packages/webob/request.py:1278: in call_application
3205    app_iter = application(self.environ, start_response)
3206../../../virtualenv/python3.6.3/lib/python3.6/site-packages/webtest/lint.py:201: in lint_app
3207    iterator = application(environ, start_response_wrapper)
3208../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/contrib/staticfiles/handlers.py:63: in __call__
3209    return self.application(environ, start_response)
3210../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/wsgi.py:157: in __call__
3211    response = self.get_response(request)
3212../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/base.py:124: in get_response
3213    response = self._middleware_chain(request)
3214../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3215    response = response_for_exception(request, exc)
3216../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3217    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3218../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3219    response = get_response(request)
3220../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3221    response = self.get_response(request)
3222../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3223    response = response_for_exception(request, exc)
3224../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3225    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3226../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3227    response = get_response(request)
3228../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3229    response = self.get_response(request)
3230../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3231    response = response_for_exception(request, exc)
3232../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3233    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3234../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3235    response = get_response(request)
3236../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3237    response = self.get_response(request)
3238../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3239    response = response_for_exception(request, exc)
3240../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3241    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3242../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3243    response = get_response(request)
3244../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3245    response = self.get_response(request)
3246../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3247    response = response_for_exception(request, exc)
3248../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3249    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3250../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3251    response = get_response(request)
3252../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3253    response = self.get_response(request)
3254../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3255    response = response_for_exception(request, exc)
3256../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3257    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3258../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3259    response = get_response(request)
3260../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3261    response = self.get_response(request)
3262../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3263    response = response_for_exception(request, exc)
3264../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3265    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3266../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3267    response = get_response(request)
3268../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3269    response = self.get_response(request)
3270../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3271    response = response_for_exception(request, exc)
3272../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3273    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3274../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3275    response = get_response(request)
3276../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3277    response = self.get_response(request)
3278../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3279    response = response_for_exception(request, exc)
3280../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3281    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3282../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3283    response = get_response(request)
3284../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3285    response = self.get_response(request)
3286../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3287    response = response_for_exception(request, exc)
3288../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3289    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3290../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3291    response = get_response(request)
3292../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3293    response = self.get_response(request)
3294../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3295    response = response_for_exception(request, exc)
3296../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3297    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3298../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3299    response = get_response(request)
3300../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/utils/deprecation.py:140: in __call__
3301    response = self.get_response(request)
3302../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:43: in inner
3303    response = response_for_exception(request, exc)
3304../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:93: in response_for_exception
3305    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
3306../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/exception.py:41: in inner
3307    response = get_response(request)
3308../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/base.py:187: in _get_response
3309    response = self.process_exception_by_middleware(e, request)
3310../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/core/handlers/base.py:185: in _get_response
3311    response = wrapped_callback(request, *callback_args, **callback_kwargs)
3312/opt/python/3.6.3/lib/python3.6/contextlib.py:52: in inner
3313    return func(*args, **kwds)
3314../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/views/decorators/csrf.py:58: in wrapped_view
3315    return view_func(*args, **kwargs)
3316../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/views/generic/base.py:68: in view
3317    return self.dispatch(request, *args, **kwargs)
3318../../../virtualenv/python3.6.3/lib/python3.6/site-packages/rest_framework/views.py:483: in dispatch
3319    response = self.handle_exception(exc)
3320../../../virtualenv/python3.6.3/lib/python3.6/site-packages/rest_framework/views.py:443: in handle_exception
3321    self.raise_uncaught_exception(exc)
3322../../../virtualenv/python3.6.3/lib/python3.6/site-packages/rest_framework/views.py:480: in dispatch
3323    response = handler(request, *args, **kwargs)
3324../../../virtualenv/python3.6.3/lib/python3.6/site-packages/rest_framework/generics.py:192: in post
3325    return self.create(request, *args, **kwargs)
3326../../../virtualenv/python3.6.3/lib/python3.6/site-packages/rest_framework/mixins.py:21: in create
3327    self.perform_create(serializer)
3328api/actions/views.py:236: in perform_create
3329    serializer.save(user=self.request.user)
3330../../../virtualenv/python3.6.3/lib/python3.6/site-packages/rest_framework/serializers.py:214: in save
3331    self.instance = self.create(validated_data)
3332api/actions/serializers.py:150: in create
3333    return target.run_reject(user, comment)
3334osf/models/mixins.py:827: in run_reject
3335    return self._run_transition(self.TriggersClass.REJECT.value, user=user, comment=comment)
3336osf/models/mixins.py:842: in _run_transition
3337    result = trigger_fn(**kwargs)
3338../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:390: in trigger
3339    return self.machine._process(func)
3340../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1114: in _process
3341    return trigger()
3342../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:408: in _trigger
3343    return self._process(event_data)
3344../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:417: in _process
3345    if trans.execute(event_data):
3346../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:271: in execute
3347    event_data.machine.callbacks(itertools.chain(self.after, event_data.machine.after_state_change), event_data)
3348../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1049: in callbacks
3349    self.callback(func, event_data)
3350../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1068: in callback
3351    func(event_data)
3352osf/utils/machines.py:296: in notify_accept_reject
3353    **context
3354website/mails/mails.py:105: in send_mail
3355    message = mail.html(**context)
3356website/mails/mails.py:67: in html
3357    return render_message(tpl_name, **context)
3358website/mails/mails.py:76: in render_message
3359    return tpl.render(**context)
3360../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/template.py:462: in render
3361    return runtime._render(self, self.callable_, args, data)
3362../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:838: in _render
3363    **_kwargs_for_callable(callable_, data))
3364../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:873: in _render_context
3365    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
3366../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:899: in _exec_template
3367    callable_(context, *args, **kwargs)
3368notify_base_mako:49: in render_body
3369    ???
3370withdrawal_request_declined_html_mako:53: in render_content
3371    ???
3372E   AttributeError: 'Undefined' object has no attribute 'fullname'
3373
```

**Comment by brianjgeiger** [comment]
Similarly, 

```py
=================================== FAILURES ===================================
3604______ TestPreprintWithdrawalRequests.test_can_reject_withdrawal_request _______
3605admin_tests/preprints/test_views.py:630: in test_can_reject_withdrawal_request
3606    response = views.PreprintRejectWithdrawalRequest.as_view()(request, guid=preprint._id)
3607../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/views/generic/base.py:68: in view
3608    return self.dispatch(request, *args, **kwargs)
3609../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/contrib/auth/mixins.py:92: in dispatch
3610    return super(PermissionRequiredMixin, self).dispatch(request, *args, **kwargs)
3611../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/views/generic/base.py:88: in dispatch
3612    return handler(request, *args, **kwargs)
3613admin/preprints/views.py:367: in post
3614    withdrawal_request.run_reject(self.request.user, withdrawal_request.comment)
3615osf/models/mixins.py:827: in run_reject
3616    return self._run_transition(self.TriggersClass.REJECT.value, user=user, comment=comment)
3617osf/models/mixins.py:842: in _run_transition
3618    result = trigger_fn(**kwargs)
3619../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:390: in trigger
3620    return self.machine._process(func)
3621../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1114: in _process
3622    return trigger()
3623../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:408: in _trigger
3624    return self._process(event_data)
3625../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:417: in _process
3626    if trans.execute(event_data):
3627../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:271: in execute
3628    event_data.machine.callbacks(itertools.chain(self.after, event_data.machine.after_state_change), event_data)
3629../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1049: in callbacks
3630    self.callback(func, event_data)
3631../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1068: in callback
3632    func(event_data)
3633osf/utils/machines.py:296: in notify_accept_reject
3634    **context
3635website/mails/mails.py:105: in send_mail
3636    message = mail.html(**context)
3637website/mails/mails.py:67: in html
3638    return render_message(tpl_name, **context)
3639website/mails/mails.py:76: in render_message
3640    return tpl.render(**context)
3641../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/template.py:462: in render
3642    return runtime._render(self, self.callable_, args, data)
3643../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:838: in _render
3644    **_kwargs_for_callable(callable_, data))
3645../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:873: in _render_context
3646    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
3647../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:899: in _exec_template
3648    callable_(context, *args, **kwargs)
3649notify_base_mako:49: in render_body
3650    ???
3651withdrawal_request_declined_html_mako:53: in render_content
3652    ???
3653E   AttributeError: 'Undefined' object has no attribute 'fullname'
3654 TestPreprintWithdrawalRequests.test_approve_reject_on_list_view[rejectRequest-rejected] 
3655admin_tests/preprints/test_views.py:673: in test_approve_reject_on_list_view
3656    response = views.PreprintWithdrawalRequestList.as_view()(request)
3657../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/views/generic/base.py:68: in view
3658    return self.dispatch(request, *args, **kwargs)
3659../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/contrib/auth/mixins.py:92: in dispatch
3660    return super(PermissionRequiredMixin, self).dispatch(request, *args, **kwargs)
3661../../../virtualenv/python3.6.3/lib/python3.6/site-packages/django/views/generic/base.py:88: in dispatch
3662    return handler(request, *args, **kwargs)
3663admin/preprints/views.py:332: in post
3664    withdrawal_request.run_reject(self.request.user, withdrawal_request.comment)
3665osf/models/mixins.py:827: in run_reject
3666    return self._run_transition(self.TriggersClass.REJECT.value, user=user, comment=comment)
3667osf/models/mixins.py:842: in _run_transition
3668    result = trigger_fn(**kwargs)
3669../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:390: in trigger
3670    return self.machine._process(func)
3671../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1114: in _process
3672    return trigger()
3673../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:408: in _trigger
3674    return self._process(event_data)
3675../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:417: in _process
3676    if trans.execute(event_data):
3677../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:271: in execute
3678    event_data.machine.callbacks(itertools.chain(self.after, event_data.machine.after_state_change), event_data)
3679../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1049: in callbacks
3680    self.callback(func, event_data)
3681../../../virtualenv/python3.6.3/lib/python3.6/site-packages/transitions/core.py:1068: in callback
3682    func(event_data)
3683osf/utils/machines.py:296: in notify_accept_reject
3684    **context
3685website/mails/mails.py:105: in send_mail
3686    message = mail.html(**context)
3687website/mails/mails.py:67: in html
3688    return render_message(tpl_name, **context)
3689website/mails/mails.py:76: in render_message
3690    return tpl.render(**context)
3691../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/template.py:462: in render
3692    return runtime._render(self, self.callable_, args, data)
3693../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:838: in _render
3694    **_kwargs_for_callable(callable_, data))
3695../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:873: in _render_context
3696    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
3697../../../virtualenv/python3.6.3/lib/python3.6/site-packages/mako/runtime.py:899: in _exec_template
3698    callable_(context, *args, **kwargs)
3699notify_base_mako:49: in render_body
3700    ???
3701withdrawal_request_declined_html_mako:53: in render_content
3702    ???
3703E   AttributeError: 'Undefined' object has no attribute 'fullname'
3704
```

**Review by brianjgeiger — APPROVED** [review]

File: website/templates/emails/withdrawal_request_declined.html.mako [hunk 2]
```diff
@@ -6,16 +6,18 @@
...
+            Dear ${contributor.fullname},
+		    <p>
             Your request to withdraw your registration <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. The registration is still publicly available on ${reviewable.provider.name}.
 			<p>
             % if notify_comment:
                 The moderator has provided the following comment:<br>
                 ${comment}
             % endif
         % else:
+            Dear ${requester.fullname},
```
Comment (jwalz) [inline website/templates/emails/withdrawal_request_declined.html.mako:19]: This email is new for Registrations Moderation, so please go ahead and make the fix to `contributor` instead of `requester` here universal

Comment (UdayVarkhedkar) [reply website/templates/emails/withdrawal_request_declined.html.mako:19]: I did this because when this email is sent as a result of preprints, there is no `contributor` attribute in the context (See https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/utils/machines.py#L289). My understanding is that preprints only have a requester in the context because because a preprint withdrawal request has a single creator who the email is sent to whereas for a registration, the withdrawal request email is sent to all contributors on the registration.

Comment (jwalz) [reply website/templates/emails/withdrawal_request_declined.html.mako:19]: Good catch!!

**Review by jwalz — APPROVED** [review]
